leadpages.net-prod.custom-domain.json

```diff
@@ -0,0 +1,17 @@
+{
+   "providerId": "leadpages.net-prod",
```

Comment:
Hi, current convention for different environments is to use same providerId and different serviceId (see hubspot.com or gvng.com).